deployment/staging_bq_dataset.txt

```diff
@@ -1 +1 @@
-google.com:datcom-store-dev.dc_kg_2020_06_30_01_02_51
\ No newline at end of file
+google.com:datcom-store-dev:dc_kg_2020_07_07_19_47_23
```

Comment:
the second colon should be "."

Reply:
String check here? ;)

Reply:
good point. This will fail the e2e sparql test as well